### PR TITLE
Provided detailed alt text for episode 6 inflammation-01.csv

### PR DESCRIPTION
**Pull request by lc822**

### CODE REVIEW

File: _episodes/06-files.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ inflammation-01.csv
 ~~~
 {: .output}
 
-![Analysis of inflammation-01.csv](../fig/03-loop_49_1.png)
+![Output from the first iteration of the for loop. Three line graphs showing the daily average, maximum and minimum inflammation over a 40-day period for the patients in inflammation-01.csv.](../fig/03-loop_49_1.png)
```
Comment (ldko) [inline _episodes/06-files.md:89]: In the lesson we try to keep the line lengths under 100 characters. I think we could split this into multiple lines like:
```
![Output from the first iteration of the for loop. Three line graphs showing the daily average,
maximum and minimum inflammation over a 40-day period for the patients in inflammation-01.csv.](
../fig/03-loop_49_1.png)
```

**Review by ldko — COMMENTED** [review]
Thanks for putting together a pull request, Lucy! I think the text works but should be split on multiple lines.

File: _episodes/06-files.md (hunk 1 above)
Comment (lc822) [reply _episodes/06-files.md:89]: Sounds good

Comment (ldko) [reply _episodes/06-files.md:89]: Ok :). If you wouldn't mind making that change when you find the time, that would be great. Thank you!

Comment (lc822) [reply _episodes/06-files.md:91]: Done, let me know if you are happy

**Review by ldko — APPROVED** [review]
Looks good to me. Thank you, Lucy!

**Pull request merged**